includes/class-gp-notifications.php

```diff
@@ -202,15 +202,15 @@ public static function get_parent_comments( int $comment_id ): array {
 	 *
 	 * @since 0.0.2
 	 *
-	 * @param array       $comments                Array with the parent comments to the posted comment.
-	 * @param string|null $email_address_to_ignore Email from the posted comment.
+	 * @param array      $comments                Array with the parent comments to the posted comment.
+	 * @param array|null $email_address_to_ignore Emails to ignore.
 	 *
 	 * @return array The emails to be notified from the thread comments.
 	 */
-	public static function get_commenters_email_addresses( array $comments, string $email_address_to_ignore = null ): array {
+	public static function get_commenters_email_addresses( array $comments, array $email_address_to_ignore = null ): array {
 		$email_addresses = array();
 		foreach ( $comments as $comment ) {
-			if ( $email_address_to_ignore !== $comment->comment_author_email ) {
+			if ( ! in_array( $comment->comment_author_email, $email_address_to_ignore ) ) {
```

Comment:
This causes a warning:
```
Warning: in_array() expects parameter 2 to be array, null given in includes/class-gp-notifications.php on line 213
```

so it should be changed like this:

```suggestion
			if ( ! $email_address_to_ignore || ! in_array( $comment->comment_author_email, $email_address_to_ignore ) ) {
```